exercises/perfect-numbers/perfect_numbers_test.go

```diff
@@ -40,3 +40,25 @@ func TestClassifiesCorrectly(t *testing.T) {
 		}
 	}
 }
+
+// Test that the kinds are not equal to each other.
+// If they are equal, then TestKind will return false positives.
+func TestClassificationsNotEqual(t *testing.T) {
+	classifications := []struct {
+		class Classification
+		val   int
+	}{
+		{ClassificationAbundant, 0},
+		{ClassificationDeficient, 1},
+		{ClassificationPerfect, 2},
+	}
+
+	for i, pair1 := range classifications {
+		for j := i + 1; j < len(classifications); j++ {
+			pair2 := classifications[j]
+			if pair1.class == pair2.class {
+				t.Fatalf("%d should not be equal to %d", pair1.val, pair2.val)
```

Comment:
given the sample solution shown in #712 

```go
package perfect

const testVersion = 1

var ErrOnlyPositive error

type Classification struct {
}

var ClassificationDeficient Classification
var ClassificationAbundant Classification
var ClassificationPerfect Classification

func Classify(n uint64) (Classification, error) {
	return ClassificationPerfect, ErrOnlyPositive
}
```

This would cause the test to output `0 should not be equal to 1`. That would be a confusing error message because zero is already not equal to one and implies something is wrong with math instead of the submitted code